common.sh

```diff
@@ -61,6 +60,13 @@ build_libfuzzer() {
   mv libFuzzer.a $LIB_FUZZING_ENGINE
 }
 
+# Uses the capability for "fsanitize=fuzzer" in the current clang
+build_fsanitize_fuzzer() {
+  CXXFLAGS="-O2 -fno-omit-frame-pointer -g -fsanitize=address,fuzzer"
```

Comment:
Actually, I think that this is the best possible option because of the default CXXFLAGS, as these would always include e.g. `fsanitize-coverage=..` unless the user defines them.